docs/src/pages/addons/addon-gallery/index.md

```diff
@@ -130,10 +130,6 @@ Add ability to customize styles in the story preview area
 
 Embed [Figma](https://figma.com) designs in a storybook panel.
 
-### [State](https://github.com/Sambego/storybook-state)
-
-Manage state inside a story. Update components when this state changes.
-
```

Comment:
Those are actually two different addons by different authors and with different APIs. To make it more visible, we can use "State by Sambego" and "State by dump247" titles

Reply:
oh, I just noticed that now, so sorry for this. 

and maybe change the description so that the two first sentence is not identical? Just a thought